api/internal/handler/route/route.go

```diff
@@ -379,12 +380,25 @@ func (h *Handler) Create(c droplet.Context) (interface{}, error) {
 		}
 	}
 
-	ret, err := h.routeStore.Create(c.Context(), input)
+	// check name existed
+	ret, err := h.routeStore.List(c.Context(), store.ListInput{
+		Predicate: func(obj interface{}) bool {
+			return obj.(*entity.Route).Name == input.Name
+		},
+	})
+	if err != nil {
+		return &data.SpecCodeResponse{StatusCode: http.StatusInternalServerError}, err
+	}
+	if ret.TotalSize > 0 {
+		return &data.SpecCodeResponse{StatusCode: http.StatusBadRequest}, errors.New("route name is existed")
```

Comment:
typo: "route name is existed" => "route name exists".

Reply:
fixed.